src/schema/nmdc.yaml

```diff
@@ -1030,6 +1034,9 @@ enums:
         is_a: organic_layer
 
 slots:
+  biomaterial_purity:
```

Comment:
this needs metadata

Reply:
it sure does; but not in this PR.  This is a direct copy of what exists currently in the schema.